test/rubygems/test_gem_gemcutter_utilities.rb

```diff
@@ -179,7 +179,7 @@ def test_sign_in_with_other_credentials_doesnt_overwrite_other_keys
   end
 
   def test_sign_in_with_bad_credentials
-    skip 'Always uses $stdin on windows' if Gem.win_platform?
+    skip 'Always uses $stdin on windows' if Gem.win_platform? && RUBY_VERSION <= '1.9.2'
```

Comment:
Please remove this line completely, Because the master branch only supports Ruby2.2+